cmd/podman/main.go

```diff
@@ -122,6 +123,11 @@ func main() {
 			Name:  "cpu-profile",
 			Usage: "path for the cpu profiling results",
 		},
+		cli.StringFlag{
+			Name:  "default-mounts-file",
```

Comment:
I don't think we need this. and I would like to make this hidden in buildah.

Reply:
I initially added this flag for testing purposes. I can make it hidden.